### ISSUE TITLE
Mac support?

### ISSUE THREAD

**Issue by mattpackwood** [j03b/language-circuitpython#1]
I get the error "Uncaught Error: Could not find module root given file: "/Users/mattpackwood/.atom/packages/atom-circuitpython/node_modules/bindings/bindings.js". Do you have a package.json file?"  When I try to install....

**Comment by jos-b** [comment]
Hm, I've been developing on Mac and I've had a few issues with the bindings system.

The error is about the bindings with C which come from the `SerialPort` module I am using. I'm going to set up a fresh Atom installation and see if I can reproduce this. If I can I may have to look into finding a different serial port module.

Cheers for reporting :+1:

**Comment by jos-b** [comment]
Okay, I've just created a brand new user on my MacBook and I could install fine from the APM.

How are you trying to install? From Git or through the APM? If using APM, are you using the old package on Atom (`atom-circuitpython`) or the new (`language-circuitpython`).

If you could get me an Atom version as well I can try reproduce in that version (I can only test in Atom Beta).

**Comment by mattpackwood** [comment]
Hi,

I am using Atom 1.37.0 on my Mac, 

I am trying to install the “language-circuitpython,

Errors I get are:
[Enter steps to reproduce:]

1. ...
2. ...

**Atom**: 1.37.0 x64
**Electron**: 2.0.18
**OS**: Mac OS X 10.14.5
**Thrown From**: 'atom-circuitpython' package vundefined


### Stack Trace

Uncaught Error: Could not find module root given file: "/Users/mattpackwood/.atom/packages/atom-circuitpython/node_modules/bindings/bindings.js". Do you have a `package.json` file? 

```
At /Users/mattpackwood/.atom/packages/atom-circuitpython/node_modules/bindings/bindings.js:211

Error: Could not find module root given file: "/Users/mattpackwood/.atom/packages/atom-circuitpython/node_modules/bindings/bindings.js". Do you have a `package.json` file? 
    at Function.getRoot (/packages/atom-circuitpython/node_modules/bindings/bindings.js:211:13)
    at bindings (/packages/atom-circuitpython/node_modules/bindings/bindings.js:82:32)
    at /packages/atom-circuitpython/node_modules/@serialport/bindings/lib/darwin.js:1:191)
    at /packages/atom-circuitpython/node_modules/@serialport/bindings/lib/darwin.js:104:3)
    at Module.get_Module._compile (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:146782)
    at Object.value [as .js] (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:150329)
    at Module.load (module.js:561:32)
    at tryModuleLoad (module.js:504:12)
    at Function.Module._load (module.js:496:3)
    at Module.require (/app.asar/static/index.js:47:45)
    at require (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:146072)
    at /packages/atom-circuitpython/node_modules/@serialport/bindings/lib/index.js:10:22)
    at /packages/atom-circuitpython/node_modules/@serialport/bindings/lib/index.js:17:3)
    at Module.get_Module._compile (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:146782)
    at Object.value [as .js] (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:150329)
    at Module.load (module.js:561:32)
    at tryModuleLoad (module.js:504:12)
    at Function.Module._load (module.js:496:3)
    at Module.require (/app.asar/static/index.js:47:45)
    at require (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:146072)
    at /packages/atom-circuitpython/node_modules/serialport/lib/index.js:2:17)
    at /packages/atom-circuitpython/node_modules/serialport/lib/index.js:17:3)
    at Module.get_Module._compile (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:146782)
    at Object.value [as .js] (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:150329)
    at Module.load (module.js:561:32)
    at tryModuleLoad (module.js:504:12)
    at Function.Module._load (module.js:496:3)
    at Module.require (/app.asar/static/index.js:47:45)
    at require (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:146072)
    at /packages/language-circuitpython/lib/board-view.js:5:45)
```

### Commands

```
     -5:34.5.0 atom-circuitpython:toggle-serial (input.hidden-input)
     -5:22.5.0 settings-view:install-packages-and-themes (atom-pane.pane)
     -0:02.3.0 core:copy (atom-notification.fatal.icon.icon-bug.native-key-bindings.has-detail.has-close.has-stack)
```

### Non-Core Packages

```
language-circuitpython 0.6.0 
```



Matt Packwood
E-mail: mattpackwood@gmail.com
Mobile / iMessage: +1-248-346-2166
Skype: mattpackwood@msn.com

**Comment by j03b** [comment]
This should hopefully be resolved with v0.7.1
